tests/update_script.prm

```diff
@@ -17,7 +17,7 @@ subsection Adiabatic conditions model
   subsection Compute profile
     set Use surface condition function = true
     subsection Surface condition function
-      set Function expression = 1e-2*t; 1613.0+1e-13*t
+      set Function expression = 1e-2/(365.25*24*60*60)*t; 1613.0+1e-13/(365.25*24*60*60)*t
```

Comment:
Interesting that we have one test that hits this case. The value we use as constant in the code is `60*60*24*365.2425`, that should fix the tests.

Also please add an entry in the directory `doc/modules/changes`, with a text like the following:
```
Changed: The time dimension of the surface condition function of the 'compute profile' adiabatic conditions plugin now respects the value of the parameter 'Use years in output instead of seconds'.

Reply:
Done. Thank you!

Reply:
Hi Rene @gassmoeller , those two testers still failed... Seems that their 'Pressure at top (Pa)' are still off. (-1.756e-09 vs -1.034e-08 ;  2.342e-09 vs 2.732e-09)